app/controllers/users_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
 
   def destroy
     @user = policy_scope(User).find(params[:id])
+    RemoveStoriesFromUserService.call(@user, @project)
```

Comment:
We shouldn't do it before we're sure the user was deleted. It should run inside a transaction together with the `policy_scope(User).delete(@user)`